ui/partials/perf/compareconfidence.html

```diff
@@ -1,3 +1,8 @@
-<span ng-if="text && value">
-  {{value|displayPrecision}} (<abbr title="Result of running t-test on base versus new result distribution: higher values indicate greater confidence that the indicated performance difference is statistically meaningful">{{text}}</abbr>)
+<span ng-if="result.confidenceText && result.confidence">
+  {{result.confidence|displayPrecision}} (<abbr title="Result of running t-test on base versus new result distribution: higher values indicate greater confidence that the indicated performance difference is statistically meaningful">{{result.confidenceText}}</abbr>)
+  <span ng-if="result.originalRuns.length < 5 || result.newRuns.length < 5"
+        class="glyphicon glyphicon-warning-sign text-warning"
+        tooltip="Need at least 5 runs for each push (base and new) to have full confidence in the comparison"
```

Comment:
It's indeed possible to have "too few" runs with non-low confidence, but it's likely to be rare (e,g, base has 3 runs: 99 100 101, and new has 3 runs: 200 205 203 so this would yield pretty high confidence level despite the less-than-5 runs).

That being said, I do agree it could be confusing if that happens.

How about displaying the warning only if the number of runs (on either base/new) is less than 5 AND the confidence level is medium or low?